hack/tls-setup/README.md

```diff
@@ -5,5 +5,28 @@ Defaults generate an ECDSA-384 root and leaf certificates for `localhost`. etcd
 **Instructions**
 
 1. Install git, go, and make
-2. Run `make` to generate the certs
-3. Run `goreman start`
+2. Amend https://github.com/coreos/etcd/blob/master/hack/tls-setup/config/req-csr.json with IP's for each node in your cluster:
```

Comment:
let's mention that IPs (3.8.121.201, 46.4.19.20) are the fake IPs that we add into the json.